x-pack/test/security_solution_endpoint_api_int/apis/metadata.ts

```diff
@@ -31,317 +32,56 @@ import { indexFleetEndpointPolicy } from '../../../plugins/security_solution/com
 export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
 
-  describe('test metadata api', () => {
-    describe('with .metrics-endpoint.metadata_united_default index', () => {
-      const numberOfHostsInFixture = 2;
-
-      before(async () => {
-        await stopTransform(getService, `${METADATA_UNITED_TRANSFORM}*`);
-        await deleteAllDocsFromFleetAgents(getService);
-        await deleteAllDocsFromMetadataDatastream(getService);
-        await deleteAllDocsFromMetadataCurrentIndex(getService);
-        await deleteAllDocsFromIndex(getService, METADATA_UNITED_INDEX);
-
-        // generate an endpoint policy and attach id to agents since
-        // metadata list api filters down to endpoint policies only
-        const policy = await indexFleetEndpointPolicy(
-          getService('kibanaServer'),
-          'Default',
-          '1.1.1'
-        );
-        const policyId = policy.integrationPolicies[0].policy_id;
-        const currentTime = new Date().getTime();
-
-        await Promise.all([
-          bulkIndex(getService, AGENTS_INDEX, generateAgentDocs(currentTime, policyId)),
-          bulkIndex(getService, METADATA_DATASTREAM, generateMetadataDocs(currentTime)),
-        ]);
-
-        // wait for latest metadata transform to run
-        await new Promise((r) => setTimeout(r, 30000));
-        await startTransform(getService, METADATA_UNITED_TRANSFORM);
-
-        // wait for united metadata transform to run
-        await new Promise((r) => setTimeout(r, 15000));
-      });
-
-      after(async () => {
-        await deleteAllDocsFromFleetAgents(getService);
-        await deleteAllDocsFromMetadataDatastream(getService);
-        await deleteAllDocsFromMetadataCurrentIndex(getService);
-        await stopTransform(getService, `${METADATA_UNITED_TRANSFORM}*`);
-        await deleteAllDocsFromIndex(getService, METADATA_UNITED_INDEX);
-      });
-
-      it('should return one entry for each host with default paging', async () => {
-        const res = await supertest
-          .post(HOST_METADATA_LIST_ROUTE)
-          .set('kbn-xsrf', 'xxx')
-          .send()
-          .expect(200);
-        const { body } = res;
-        expect(body.total).to.eql(numberOfHostsInFixture);
-        expect(body.hosts.length).to.eql(numberOfHostsInFixture);
-        expect(body.request_page_size).to.eql(10);
-        expect(body.request_page_index).to.eql(0);
-      });
+  describe('test metadata apis', () => {
```

Comment:
same as other tests, mostly just nesting/moving tests around followed by duplicating POST API tests and modifying for the new GET API.